storybook/storybook-docs/src/grid.stories.mdx

```diff
@@ -119,15 +162,11 @@ Op precies die breedtes is de maatvoering als volgt.
 
 #### Compact
 
-| Klasse apparaat | Referentie-breedte | Aantal kolommen | Breedte kolom | Breedte witruimte | Breedte marge | Breedte grid |
-| :-------------- | -----------------: | --------------: | ------------: | ----------------: | ------------: | -----------: |
-| telefoon        |                320 |               4 |            60 |                16 |            24 |          288 |
-| tablet          |                832 |               8 |            86 |                16 |            16 |          800 |
-| desktop         |               1600 |              12 |            98 |                24 |            24 |         1440 |
-
-Let op: vanaf een vensterbreedte van 1088 pixels staat er een navigatiebalk links van het grid.
-Deze is 112 pixels breed.
-Vandaar de afwijkende breedte van het grid.
+| Klasse apparaat | Referentie-breedte | Aantal kolommen | Breedte menubalk | Breedte kolom | Breedte witruimte | Breedte marge | Breedte grid |
+| :-------------- | -----------------: | --------------: | ---------------: | ------------: | ----------------: | ------------: | -----------: |
+| telefoon        |                320 |               4 |                0 |            60 |                16 |            16 |          288 |
```

Comment:
It would help if we have an example (built in) framesize for figma, not the actual breakpoint. So for example 375x667 (iphone 8) and 834x1196 (11 inch ipad). I've played around with the tablet size and turning it to landscape will result in the desktop layout in the scenario. Perhaps we want a landscape media query in some cases?

Reply:
We want to stay away from specific screen dimensions as there are too many of them. That’s why we chose to design at 320, 832 and 1600 to represent the entire classes of devices. As for height, all designs in Figma I have seen are as tall as their content.